sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobSetHttpHeadersHeaders.java

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.azure.android.core.util.DateTimeRfc1123;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import org.threeten.bp.OffsetDateTime;
+
+/**
+ * Defines headers for SetHTTPHeaders operation.
+ */
+@JacksonXmlRootElement(localName = "Blob-SetHttpHeaders-Headers")
+public final class BlobSetHttpHeadersHeaders {
```

Comment:
I understand this is a special case but I can't help but think the name of this class is weird, heh :P

Reply:
It is weird! haha